src/fiskaltrust.Launcher/Commands/HostCommand.cs

```diff
@@ -33,197 +33,176 @@ public HostCommand() : base("host")
+                var plebeianConfiguration = PlebeianConfiguration.Deserialize(System.Text.Encoding.UTF8.GetString(Convert.FromBase64String(hostOptions.PlebeianConfiguration)));
+                
+                var cashboxConfiguration = CashBoxConfigurationExt.Deserialize(await File.ReadAllTextAsync(launcherConfiguration.CashboxConfigurationFile!));
+                cashboxConfiguration.Decrypt(launcherConfiguration, await CommonHandler.LoadCurve(launcherConfiguration.CashboxId.Value, launcherConfiguration.AccessToken!, launcherConfiguration.ServiceFolder!));
 
-            var plebeianConfiguration = Configuration.PlebeianConfiguration.Deserialize(System.Text.Encoding.UTF8.GetString(Convert.FromBase64String(hostOptions.PlebeianConfiguration)));
-
-            var cashboxConfiguration = CashBoxConfigurationExt.Deserialize(await File.ReadAllTextAsync(launcherConfiguration.CashboxConfigurationFile!));
-
-            cashboxConfiguration.Decrypt(launcherConfiguration, await CommonHandler.LoadCurve(launcherConfiguration.CashboxId!.Value, launcherConfiguration.AccessToken!, launcherConfiguration.ServiceFolder!, launcherConfiguration.UseLegacyDataProtection!.Value));
+                var packageConfiguration = (plebeianConfiguration.PackageType switch
+                {
+                    PackageType.Queue => cashboxConfiguration.ftQueues,
+                    PackageType.SCU => cashboxConfiguration.ftSignaturCreationDevices,
+                    PackageType.Helper => cashboxConfiguration.helpers,
+                    _ => throw new Exception($"Unknown PackageType {plebeianConfiguration.PackageType}")
+                }).First(p => p.Id == plebeianConfiguration.PackageId);
 
-            var packageConfiguration = (plebeianConfiguration.PackageType switch
-            {
-                PackageType.Queue => cashboxConfiguration.ftQueues,
-                PackageType.SCU => cashboxConfiguration.ftSignaturCreationDevices,
-                PackageType.Helper => cashboxConfiguration.helpers,
-                var unknown => throw new Exception($"Unknown PackageType {unknown}")
-            }).First(p => p.Id == plebeianConfiguration.PackageId);
+                packageConfiguration.Configuration = ProcessPackageConfiguration(packageConfiguration.Configuration, launcherConfiguration, cashboxConfiguration);
 
-            packageConfiguration.Configuration = ProcessPackageConfiguration(packageConfiguration.Configuration, launcherConfiguration, cashboxConfiguration);
+                IProcessHostService? processHostService = null;
+                if (!hostOptions.NoProcessHostService)
+                {
+                    string grpcAddress = launcherConfiguration.LauncherServiceUri?.ToString();
+                    processHostService = GrpcChannel.ForAddress(grpcAddress).CreateGrpcService<IProcessHostService>();
 
```

Comment:
We don't use grpc over http anymore. You'll need to create a GrpcChannel like described

here https://learn.microsoft.com/en-us/aspnet/core/grpc/interprocess-namedpipes?view=aspnetcore-8.0#client-configuration on windows.

and here https://learn.microsoft.com/en-us/aspnet/core/grpc/interprocess-uds?view=aspnetcore-8.0#client-configuration on linux.